### DISCUSSION TITLE
New to hammerspoon - how to do this config? window switcher

### DISCUSSION THREAD

**Discussion by tb-1212** [Hammerspoon/hammerspoon#3391] [Q&A]
hi I'm new to hammerspoon and need some help with this, the window switcher is working however I'm not quite sure where to place code for ui so its not showing thumbnails, can someone tell me where to place the code? and how can i exclude certain apps in the switcher? 

Really appreciate the help

![image](https://user-images.githubusercontent.com/119829550/221477604-9d5ac98c-a632-4dad-ab79-cf5418a8914d.png)

**Comment by igorepst** [comment]
First of all, you don't need the first line, as you only ever use `switcher_space` and `switcher_browsers`. Now, regarding your question, it is simply `switcher_space.ui.showThumbnails = false`
